src/Glpi/Asset/Capacity/HasVirtualMachineCapacity.php

```diff
@@ -51,6 +52,12 @@ public function getIcon(): string
         return ItemVirtualMachine::getIcon();
     }
 
+    #[Override]
+    public function getDescription(): string
+    {
+        return __("List virtual machines attached to this asset and found by automatic inventory");
```

Comment:
Not only from inventory
```suggestion
        return __("List virtual machines attached to this asset");
```